### PR TITLE
Style guide enhancements

### PR DESCRIPTION
**Pull request by cpholguera**
This PR enhances the style guide.

- enhanced abbreviations section
- using mobile app related terms in examples (e.g. analyze, encryption) instead of unrelated ones (e.g. torch, walk)
- new web page for capitalization (previous is not available)
- added note about file extensions
- added more common contractions
- fixed: dex -> DEX, apk/Apk/.apk -> APK, ipa/.ipa -> IPA, jar -> JAR, MacOSX -> macOS, zip -> ZIP

### CODE REVIEW

**Comment by OMTGreleaser** [comment]
Results for commit 8e847145f0fff5f272a0f2f767fd525a18e90758: Broken link result: [!] Error(s) found by the Links validator: 6 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        0 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

File: style_guide.md [hunk 1]
```diff
@@ -90,24 +90,24 @@ When using any number higher than ten, use the numeric version (e.g., "12" or 
...
 We follow the title case rules from the "Chicago Manual of Style":  
 
 - Capitalize the first and last word in a title, regardless of part of speech
-- Capitalize all nouns (baby, country, picture), pronouns (you, she, it), verbs (walk, think, dream), adjectives (sweet, large, perfect), adverbs (immediately, quietly), and subordinating conjunctions (as, because, although)
+- Capitalize all nouns (app, encryption, package), pronouns (you, she, it), verbs (analyze, compile, inspect), adjectives (active, insecure, weak), adverbs (immediately, quietly), and subordinating conjunctions (as, because, although)
 - Lowercase “to” as part of an infinitive
 - Lowercase all articles (a, the), prepositions (to, at, in, with), and coordinating conjunctions (and, but, or)
 
-When in doubt, you can verify proper capitalization on [www.titlecapitalization.com](http://www.titlecapitalization.com/).
+When in doubt, you can verify proper capitalization on [https://titlecaseconverter.com/](https://titlecaseconverter.com/).
```
Comment (TheDauntless) [inline style_guide.md]: Add alt-text
```suggestion
When in doubt, you can verify proper capitalization on [https://titlecaseconverter.com/](https://titlecaseconverter.com/ "Title Case Converter").
```

File: style_guide.md [hunk 2]
```diff
@@ -140,9 +142,26 @@ you will -> you'll
...
 ### Abbreviations
 
-Spell out abbreviations the first time they are used.
+Abbreviations include acronyms, initialisms, shortened words, and contractions.
 
-Ex: United States (U.S.)
+- Spell out the term the first time it's used, followed by the abbreviation in parentheses. Example: Mobile Security Testing Guide (MSTG). Subsequent usages in the same chapter may include the abbreviation only.
+- If it only appears once in the content, spell out the term instead of using the abbreviation.
+- In titles and headings, use the abbreviation but be sure to properly introduce it (see above) in the text that follows.
+- Use "a" or "an" depending on the pronunciation (as with regular words). Example: a DLL, an APK, a URL.
```
Comment (TheDauntless) [inline style_guide.md]:
```suggestion
- Use "a" or "an" depending on the pronunciation of the acronym. Example: a DLL, an APK, a URL, a SQL.
```

I don't remember which one, but there are a few that have a different pronunciation depending on if you say the abbreviation or the full words. Let's make the acronym the default one.

I also added the SQL example as it should be pronounced "sequel", but many people pronounce it as es-que-el.

**Comment by OMTGreleaser** [comment]
Results for commit f9d4f454990f2133525975fb287fa138a611d8d8: Broken link result: [!] Error(s) found by the Links validator: 3 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        0 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Comment by OMTGreleaser** [comment]
Results for commit a03553f8bc023f7432271ef3e69cff742576855e: Broken link result: [!] Error(s) found by the Links validator: 3 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        1 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Comment by OMTGreleaser** [comment]
Results for commit 7674b6fa4c4e5d8fbebe4e57b7e91f5647b93f09: Broken link result: [!] Error(s) found by the Links validator: 3 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        1 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Comment by OMTGreleaser** [comment]
Results for commit 0f4ecd021192a8198659540f9127566902b97959: Broken link result: [!] Error(s) found by the Links validator: 3 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        0 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Pull request merged**